solutions/USACO dec19-gymnastics.mdx

```diff
@@ -68,4 +68,53 @@ public class gymnastics {
 ```
 </JavaSection>
 
-</LanguageSection>
\ No newline at end of file
+<CPPSection>
+
+## Alternate Solution
+
+We generate the ranking as we read the data.
+
+We create a 2D array of booleans which states if $A$ has a ranking higher than $B$ in any point of time.
+We read every ranking, then for all $\frac{n(n+1)}{2}$ pairs we set $\texttt{b}[A][B]=\text{true}$.
+
+Then we iterate though all pairs in $\mathcal{O}(N^2)$ time and check if they satisfy the condition. At least one of $\texttt{b}[A][B]$ and $\texttt{b}[B][A]$ must be true, and hence we only need to check if they are both true. If one of them is false, we increment our $\texttt{count}$ by 1.
```

Comment:
don't use \texttt around single-letter variable names. Why is N capitalized here but not capitalized above?